src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -454,7 +472,18 @@ private static void fillMethodModel(MethodModel mModel, String name, ArrayList<S
                         break;
                 }
                 if (j < comment.length()) {
-                    mModel.setDescription(mModel.getDescription() + ' ' + comment.substring(j));
+                    if (inDescription) {
+                        mModel.setDescription(mModel.getDescription() + ' ' + comment.substring(j));
+                    } else if (inExample) {
```

Comment:
why is the example code different than the description code?  are you trying to support @example repeated on multiple lines?  is that something javadoc does?